### PR TITLE
corrected update_network() and added update_config()

### PR DESCRIPTION
**Pull request by Jumper78**
update_network() did not work for me anymore since the data structure was not complete; for my Indego (Indego S+ 400), there are additional parameters (currMode, configMode, ..., networks) that are transmitted from the cloud

update_config() had been kicked out after the re-structuring of the code; I added the necessary functions and data structure

both functions work for me but seem to cause problems with other Indegos

### CODE REVIEW

**Comment by jm-73** [comment]
@eavanvalkenburg Anything to add to this PR?

**Comment by eavanvalkenburg** [comment]
no go ahead and merge, but hold off on the release, I will rebase with these changes and then submit mine

File: pyIndego/indego_client.py [hunk 1]
```diff
@@ -137,6 +137,10 @@ def update_network(self):
         """Update network."""
         self._update_network(self.get(f"alms/{self._serial}/network"))
 
+    def update_config(self):
```
Comment (eavanvalkenburg) [inline pyIndego/indego_client.py:140]: Please implement it the same way but with await before self.get in indego_async_client

**Review by eavanvalkenburg — CHANGES_REQUESTED** [review]
I miss the new update_config method in the indego async client, should be the same as the one in the indego_client just with async def and await before self.get

**Comment by Jumper78** [comment]
Hello @eavanvalkenburg ,

thank you for the hint. I am not yet familiar enough with the new structure an have not been aware that the async part might also need the function.

I will have a look tonight.

**Comment by eavanvalkenburg** [comment]
> Hello @eavanvalkenburg ,
> 
> thank you for the hint. I am not yet familiar enough with the new structure an have not been aware that the async part might also need the function.
> 
> I will have a look tonight.

No problem at all, thanks for contributing!

**Comment by jm-73** [comment]
Should I scrap this PR and wait for a new one? @Jumper78 @eavanvalkenburg

**Comment by eavanvalkenburg** [comment]
No just wait for @Jumper78 to push additional code to this pr

**Comment by eavanvalkenburg** [comment]
Lgtm

**Comment by Jumper78** [comment]
Wait a second! I just found a tiny mistake in README.md. I will commit again in a second.

**Comment by Jumper78** [comment]
Now I am happy with the changes. @jm-73, you can go ahead with the pull request

**Pull request merged**